doc/source/reference/random/examples/cython/index.rst

```diff
@@ -6,6 +6,6 @@ Extending `numpy.random` via Cython
 
 
 .. toctree::
-    setup.py.rst
+    messon.build.rst
```

Comment:
Being a bit nitpicky about the file name here: 
```suggestion
    meson.build.rst
```
but also, I believe since this is still in transition, we may want to add a note that meson is the new build system, for those expecting a setup.py file? Something like

````
.. note::

Starting with NumPy 1.26.0, Meson is the default build system for NumPy.
See :ref:`distutils-status-migration`.

````

Reply:
Done